precipitation/ngsild-payloads/Forecast.json

```diff
@@ -0,0 +1,191 @@
+        "value":10.0,
+        "unitCode":"CEL",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:temperature:CEL:MIN:APIX"
+    },
+    {
+        "type": "Property",
+        "value":20.0,
+        "unitCode":"CEL",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:temperature:CEL:MAX:APIX"
+    },
+    {
+        "type": "Property",
+        "value":15.0,
+        "unitCode":"CEL",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:temperature:CEL:AVG:APIX"
+    }
+    ],
+    "windDirection":[
+        {
+        "type": "Property",
+        "value":220,
+        "unitCode":"DEG",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+    }
+],
+    "windSpeed":[    
+        {
+        "type": "Property",
+        "value":50,
+        "unitCode":"KMH",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:gust:KMH:APIX"
+    },
+    {
+        "type": "Property",
+        "value":20,
+        "unitCode":"KMH",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:wind:KMH:AVG:APIX"
+    }
+],
+    "windProb":[    
+        {
+        "type": "Property",
+        "value":40,
+        "unitCode":"P1",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:prob70:P1:APIX"
+    },
+    {
+        "type": "Property",
+        "value":10,
+        "unitCode":"P1",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:prob100:P1:APIX"
+    }
+],
+    "sunHours":
+    { 
+        "type": "Property",
+        "value":7,
+        "unitCode":"H",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
```

Comment:
this is not follwing the recommendation  of my last comment! In which I have recommended to support the datasetId even with one value.
Also this will generate a json format error.
Use visual studio text editor it will support checking json format error